native/packages/react-native-bpk-theming/src/BpkThemeProvider.js

```diff
@@ -16,11 +16,70 @@
  * limitations under the License.
  */
 
- import { ThemeProvider, withTheme } from 'theming';
+ import React from 'react';
+ import PropTypes from 'prop-types';
+ import { ThemeProvider } from 'theming';
+
+ // This is the same as https://github.com/iamstarkov/theming/blob/master/src/create-theme-listener.js
```

Comment:
I think it's worth discuss with @matthewdavidson and @k0nserv about this approach, I personally think is the best compromise of going around an issue that the 3rd party library is causing without forking the library or changing to something else, and if the main library get update we can remove this

What do you guys think?

Reply:
I agree with this approach 👍